lib/chusaku.rb

```diff
@@ -80,14 +80,31 @@ def clean_group(group)
     def annotate_group(group:, route_data:)
       whitespace = /^(\s*).*$/.match(group[:body])[1]
       route_data.reverse_each do |datum|
-        name = datum[:name]
-        annotation = "@route #{datum[:verb]} #{datum[:path]}"
-        annotation += " (#{name})" unless name.nil?
-        comment = "#{whitespace}# #{annotation}\n"
+        comment = "#{whitespace}# #{annotate_route(**datum)}\n"
         group[:body] = comment + group[:body]
       end
     end
 
+    # Generate route annotation
+    #
+    # @param {String} verb
+    # @param {String} path
+    # @param {String} name
+    # @param {Hash} defaults
+    # @return {String}
```

Comment:
🙇🏻 Appreciate you tacking this on. Would you mind giving a brief description for each of the parameters as well?

```suggestion
    # Generate route annotation.
    #
    # @param {String} verb - HTTP verb for route
    # @param {String} path - Rails path for route
    # @param {String} name - Name used in route helpers
    # @param {Hash} defaults - Default parameters for route
    # @return {String} - @route <verb> <path> {<defaults>} (<name>)
```